themes/digital.gov/layouts/partials/core/guides/guide-nav.html

```diff
@@ -1,88 +1,86 @@
-<aside class="guide-nav-box usa-layout-docs__sidenav">
-  <nav aria-label="Secondary navigation">
-    <ul class="usa-sidenav guide-nav">
-      {{- $current_page := . -}}
+<nav aria-label="Secondary navigation">
+  <ul class="usa-sidenav dg-guide__nav-list">
+    {{- $current_page := . -}}
 
-      {{/* Guide YML file (optional way of configuring the nav) */}}
-      {{- $guide_data := index $.Site.Data.guidenav (.Params.guide) -}}
-      {{/* All the guide files */}}
-      {{- $guide_files := (where .Site.Pages ".Params.guide" .Params.guide) -}}
-      {{/* Check for a guide YML file */}}
-      {{- if and $guide_data $guide_data.nav -}}
-        {{/* {{ template "guidenav-data-driven" (dict "current_page" $current_page "guide_data" $guide_data.nav "guide_files" $guide_files ) }}
-        */}}
-        {{/* Gets the URL of the current page in the guide */}}
-        {{- $current_url := $current_page.Permalink -}}
+    {{/* Guide YML file (optional way of configuring the nav) */}}
+    {{- $guide_data := index $.Site.Data.guidenav (.Params.guide) -}}
+    {{/* All the guide files */}}
+    {{- $guide_files := (where .Site.Pages ".Params.guide" .Params.guide) -}}
+    {{/* Check for a guide YML file */}}
+    {{- if and $guide_data $guide_data.nav -}}
+      {{/* Gets the URL of the current page in the guide */}}
+      {{- $current_url := $current_page.Permalink -}}
 
-        {{- range $guide_data.nav -}}
-          {{- $item := . -}}
-          {{- $page := $.Site.GetPage $item.path }}
-          <li class="usa-sidenav__item">
-            <a
-              class="{{- if eq $current_url $page.Permalink -}}
-                usa-current
-              {{- end -}}"
-              href="{{- $page.Permalink -}}"
-              title="{{- $item.title -}}"
-            >
-              {{- $item.title }}
-              {{ if $item.subnav -}}<span class="arrow-right"></span>{{- end -}}
-            </a>
+      {{- range $guide_data.nav -}}
+        {{- $item := . -}}
+        {{- $page := $.Site.GetPage $item.path }}
+        <li class="usa-sidenav__item">
+          <a
+            class="{{- if eq $current_url $page.Permalink -}}
+              usa-current
+            {{- end -}}"
+            href="{{- $page.Permalink -}}"
+            title="{{- $item.title -}}"
+          >
```

Comment:
Heads up, this outputs an empty class regular links. Because of the original way this was coded it appears multiple times in template.

<img width="500" alt="image" src="https://github.com/GSA/digitalgov.gov/assets/3385219/84a38d41-0cc8-4909-8838-f178e9266c63">

Reply:
Yes, I did notice this. I've added this detail to an existing trello card.